.github/workflows/CI.yml

```diff
@@ -0,0 +1,31 @@
+name: CI
+on: [push]
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        julia-version: ['~1.8', '~1.9']
+        os: [ubuntu-latest, macOS-latest]
```

Comment:
do we really need mac here? it tends to be slower and more expensive

Reply:
I do want it to work on mac as well as linux. I wouldn't say we really need it, though, since we don't do anything fancy ourselves, that's done by the dependencies, mainly the sdk. I'm fine with removing it if it's a concern.